src/data/links/2020-03-23_impossible_foods.yaml

```diff
@@ -10,5 +10,5 @@ countries:
   - hk
   - sg
 description: >-
-  We’re making meat from plants so that we never have to use animals again.  Just one Impossible Burger (instead of a burger made from cows) will use 96% less land, 87% less water, and 89% fewer GHG emissions.
+  We’re making meat from plants so that we never have to use animals again.  Just one Impossible Burger (instead of a burger made from cows) will save the equivalent of: 96% less land, 87% less water, and 89% fewer GHG emissions.
```

Comment:
```suggestion
  We’re making meat from plants so that we never have to use animals again.  Just one Impossible Burger (instead of a burger made from cows) will use 96% less land, 87% less water, and 89% fewer GHG emissions.
```